includes/gateway-settings-komoju.php

```diff
@@ -0,0 +1,32 @@
+<?php
```

Comment:
This file is copy/paste from the old 'settings-komoju.php'. I just changed the name and removed the general stuff into a different file.